deploy-web/src/components/layout/WelcomeModal.tsx

```diff
@@ -1,5 +1,5 @@
 import { Button, Dialog, DialogContent, DialogActions, Typography, Box, Paper, useTheme } from "@mui/material";
-import { useKeplr } from "@src/context/KeplrWalletProvider";
+import { useKeplr } from "@src/context/WalletProvider";
```

Comment:
```suggestion
```
Should be `useWallet`, but it's not used anymore so could just remove the import

Reply:
I have made the changes.